.upptimerc.yml

```
@@ -1,31 +1,37 @@
-# Change these first
-owner: upptime # Your GitHub organization or username, where this repository lives
-repo: upptime # The name of this repository
-
+owner: okp4
+repo: uptime
 sites:
-  - name: Google
-    url: https://www.google.com
-  - name: Wikipedia
-    url: https://en.wikipedia.org
-  - name: Hacker News
-    url: https://news.ycombinator.com
-  - name: Test Broken Site
-    url: https://thissitedoesnotexist.koj.co
-
+  - name: OKP4 Web Site
+    url: 'https://okp4.network'
+  - name: OKP4 Documentation Site
+    url: 'https://docs.okp4.network'
+  - name: OKP4 Faucet
+    url: 'https://faucet.okp4.network'
+  - name: OKP4 Explorer
+    url: 'https://explore.okp4.network'
+  - name: Nemeton Program Web Site
+    url: 'https://nemeton.okp4.network'
+  - name: Network okp4-nemeton-1
```

```suggestion
  - name: RPC okp4-nemeton-1
```

Maybe we can call it `RPC` instead? This node doesn't responding doesn't mean the entire network is down

---

Yes. Good suggestion.